proto/folio/tax_fee_quote.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package holms.types.folio;
+option csharp_namespace = "HOLMS.Types.Folio";
+option go_package = "folio";
+
+import "money/accounting/tax_fee.proto";
+import "primitive/monetary_amount.proto";
+
+message TaxFeeQuote {
```

Comment:
This name is awkward. I think this is the amount of tax assessed on something (e.g. folio)? You might call this `AssessedTaxFeeAmount` or `ChargedTaxFeeAmount`. If you want to stick with quote, call it `QuotedTaxFeeAssessment`. I think you should use the word "assessment" though, as it's something calculated relative to something else.

Reply:
As I think about it, we could use a more fundamental overhaul of tax/fees. This sounds like a lot of work but actually it isn't. 

- Rather than this idiotic tax/fee nomenclature, we can just have the concept of an "Assessment"
- Assessment is an umbrella concept over taxes and fees. Roll all of them into this. 
- As assessment has a bunch of "subjects", which are charge types: incidentals, misc, lodging, etc. - bools.
- An Assessment has an assessment type: ad vaolrem (% of something), fixed charge, etc.
- As assessment could have one of several selectable "exemptions" where we model the vagaries of the tax code: stays over N days, etc. 

What do you think? If you like this, let's chore it out. I've never liked this "tax/fee" nomenclature.